src/libs/actions/Report.js

```diff
@@ -1269,6 +1269,29 @@ function saveReportActionDraft(reportID, reportActionID, draftMessage) {
     Onyx.set(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS_DRAFTS}${reportID}_${reportActionID}`, draftMessage);
 }
 
+/**
+ * Syncs up a chat report and an IOU report in Onyx after an IOU transaction has been made
+ * and they've both been updated in the back-end.
+ *
+ * @param {Object} chatReportStuff
+ * @param {Object} iouReportStuff
+ */
+function syncChatAndIOUReports(chatReportStuff, iouReportStuff) {
+    const iouReportData = {};
+    const chatReportData = {};
+    const chatReportKey = `${ONYXKEYS.COLLECTION.REPORT}${chatReportStuff.reportID}`;
+    const iouReportKey = `${ONYXKEYS.COLLECTION.REPORT_IOUS}${iouReportStuff.reportID}`;
+
+    chatReportData[chatReportKey] = getSimplifiedReportObject(chatReportStuff);
+    chatReportData[chatReportKey].iouReportID = iouReportStuff.reportID;
```

Comment:
I think this key should not be set if the linkedIOU report is closed (after PayIOU for example).

Reply:
Not too sure what you mean here by linkedIOU report. Are you referring to the iouReport that already exists in Onyx?

Reply:
>  Are you referring to the iouReport that already exists in Onyx?

Yep. Currently this is not a problem because we use `hasOutstandingIOU`, but really the chatReport should no longer be linked to the paid iouReport. It's not accurate and potentially confusing. 

But I just realised that this change wasn't introduced by your PR, so no need to make this change now.